backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalRepository.java

```diff
@@ -68,4 +69,11 @@ public SAPGLJournal create(
 		final SAPGLJournalLoaderAndSaver loaderAndSaver = new SAPGLJournalLoaderAndSaver();
 		return loaderAndSaver.create(createRequest, currencyConverter);
 	}
+
+	public void setDocStatus(final @NonNull SAPGLJournalId glJournalId, final String code)
+	{
+		final I_SAP_GLJournal sourceRecord = InterfaceWrapperHelper.load(glJournalId, I_SAP_GLJournal.class);
+		sourceRecord.setDocStatus(DocStatus.Reversed.getCode());
+		InterfaceWrapperHelper.save(sourceRecord);
+	}
```

Comment:
❗ input parameter `String code` is not used in this function

evermore, we can drop the method and use existing `SAPGLRepository.updateById`

Reply:
correction done